zeek/plugin/include/zeek-compat.h

```diff
@@ -217,7 +225,11 @@ inline auto ZeekArgs_Get(const std::vector<::zeek::TypePtr>& vl, uint64_t idx) {
 inline auto event_mgr_Enqueue(const ::zeek::EventHandlerPtr& h, ::zeek::Args vl) {
     return ::zeek::event_mgr.Enqueue(h, std::move(vl));
 }
+#if ZEEK_VERSION_NUMBER >= 40000 // Zeek >= 4.0
```

Comment:
Isn't this a recent master change after the 4.0 cut, hence the check would be for >= 4.1?

Reply:
4.0 is the earliest time in which the namespaced-global is available for use.  4.1 is latest time at which people must start using the namespaced-global because the (deprecated) non-namespaced-global has been removed and will cause breakages.